package.json

```diff
@@ -85,8 +85,10 @@
         "test": "NODE_ICU_DATA=node_modules/full-icu BABEL_ENV=test NODE_ENV=test yarn jest -c scripts/jest/jest.config.js",
         "test:e2e": "BROWSERSLIST_ENV=test npm-run-all -p -r start:examples cy:run",
         "test:e2e:open": "BROWSERSLIST_ENV=test npm-run-all -p -r start:examples cy:open",
-        "test:images": "yarn test -c scripts/jest/jest.images.config.js --maxWorkers=4",
+        "test:images": "NODE_ICU_DATA=node_modules/full-icu BABEL_ENV=test NODE_ENV=test yarn jest -c scripts/jest/jest.images.config.js --maxWorkers=4",
+        "test:images:snapshots": "NODE_ICU_DATA=node_modules/full-icu BABEL_ENV=test NODE_ENV=test yarn jest -c scripts/jest/jest.images.config.js -u --maxWorkers=4",
```

Comment:
Can we change the name of the `:snapshots` variants to `:update`?

E.g.:
- `"test:images:update"`
- `"test:visuals:update"`